.github/actions/setup-greptimedb-cluster/action.yml

```diff
@@ -58,17 +58,22 @@ runs:
   - name: Wait for GreptimeDB
     shell: bash
     run: |
+      sleep 5s
       kubectl get pods -n my-greptimedb
       kubectl wait \
```

Comment:
Doesn't `kubectl wait` wait enough time?

Reply:
`kubectl wait` can't wait for non-existent resources. So it must sleep for seconds...